Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -835,7 +849,14 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                     break;
             }
             
-            securityStatusCell.imageView.image = [EncryptionTrustLevelBadgeImageHelper userBadgeImageFor:self.encryptionTrustLevel];
+            if (self.mxRoom.summary.isEncrypted)
```

Comment:
Same remark:
If `self.encryptionTrustLevel` this is not needed to check that as `EncryptionTrustLevelBadgeImageHelper: userBadgeImageFor:` will return nil for `UserEncryptionTrustLevelNone`

Reply:
You right, updated.